### ISSUE TITLE
 `ready` event for BrowserWindow doesn't exist.

### ISSUE THREAD

**Issue by ali123plus** [bradtraversy/electron-course-files#19]
Hi

i think in the image-shrink/main.js file it should be 
`mainWindow.on('closed', () => (mainWindow = null))`

according to documentation `ready` event for BrowserWindow doesn't exist.

**Comment by bradtraversy** [comment]
You are correct. In the course for this project, I added a tooltip. I just updated the repo as well. thanks
